spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -1,11 +1,11 @@
-# Sort these entries via IntelliJ's "Edit" -> "Sort Lines".
+# Sort the entries below via IntelliJ's "Edit" -> "Sort Lines".
```

Comment:
I'm surprised these were not sorted already. Is this maybe locale dependent? @sschuberth have you sorted these entries when you recently created this file?

Reply:
ok, great - all fine then. Thanks!